samples/nrf9160/modem_shell/README.rst

```diff
@@ -379,6 +379,18 @@ Building and running
 
 .. include:: /includes/build_and_run_nrf9160.txt
 
+DK buttons
+==========
+
+The buttons have the following functions:
+
+Button 1:
+    * Pressing of the button: raising a kill/abort signal. I.e. long enough press of the button will kill/abort all supported running commands.
```

Comment:
I would add list of supported commands. Something like:
"The following commands can be aborted: iperf3, curl, sock send -l"

Reply:
Updated (+ping and th).